### PR TITLE
Build galactic, humble and humble-devel images

### PR DESCRIPTION
**Pull request by domire8**
Okay so this is kinda the 'release' of humble. We replace the galacitc-devel image with a humble-devel image. This means:
- Galactic image with main branch of control libraries, galactic branch of modulo (which is equivalent to v2.1.1)
- Humble image with main branch of control libraries, main branch of modulo
- Humble-devel image with main branch of control libraries, humble branch of modulo (TODO!! this shall be changed to develop once we merged humble into develop on modulo)

I'm currently running this here: https://github.com/domire8/docker-images/actions/runs/3344142806

### CODE REVIEW

**Review by buschbapti — APPROVED** [review]
This pipeline is getting nicely

File: .github/workflows/check-upstream-devel.yml
Comment (eeberhard) [inline .github/workflows/check-upstream-devel.yml:21]: I think renaming these will also be part of the TODO you mentioned (when modulo develop branch officially supports humble). But for now it's more clear to keep them named like this.

Either it becomes `cl_develop_id` and `modulo_develop_id`, or back to just `cl_id` and `modulo_id`, whichever is cleaner. Since they are just output flags, the latter is probably better and more consistent with the other workflow.

**Review by eeberhard — APPROVED** [review]
Looks good!

File: .github/workflows/check-upstream-devel.yml
Comment (domire8) [reply .github/workflows/check-upstream-devel.yml:21]: Indeed yeah, and good suggestion :+1:

**Comment by domire8** [comment]
https://github.com/domire8/docker-images/actions/runs/3344668867

This seems to work, I will merge once control libraries patch release is merged as well, to save at least some computation

**Pull request merged**
